test/distributed/test_c10d_nccl.py

```diff
@@ -2926,7 +2907,7 @@ def tearDown(self):
 
     @property
     def op_timeout_sec(self):
-        return 1
+        return 3
```

Comment:
Wut

Reply:
Just to increase the timeout limit by a little to avoid a false timeout, so that the chance of being flaky is smaller.

Reply:
is this gonna sleep for 3x longer or is it a nice timeout that only waits that long if needed?  i'm a little mindful of our reputation for too-long CI jobs..

Reply:
"a nice timeout that only waits that long if needed"